Sources/Nuke/Internal/Mutex.swift

```diff
@@ -4,11 +4,11 @@
 
 import Foundation
 
-final class Atomic<T>: @unchecked Sendable {
```

Comment:
I assume that it would be good to take [Mutex](https://github.com/pointfreeco/swift-sharing/blob/main/Sources/Sharing/Internal/MutexBackport.swift) backport implementation from `swift-sharing` framework, which is ported from `Swift.Synchronization`

Reply:
Hmm, interesting. Thanks for the link. I'll look into it.